tests/kernel_performance/install_qatestset.pm

```diff
@@ -169,24 +170,40 @@ sub os_update {
     zypper_call("dup", timeout => 1800);
 }
 
+sub set_selinux {
+    return unless (has_selinux);
+
+    record_info('sestatus', script_output('sestatus'));
+    my $selinux_mode = get_var('HANAPERF_SELINUX_SETENFORCE', 'Enforcing');
+
+    return if (script_output('getenforce') =~ m/$selinux_mode/i);
+
+    if ($selinux_mode =~ m/disabled/i) {
+        replace_grub_cmdline_settings('security=selinux', 'security=apparmor', update_grub => 0);
```

Comment:
I am not sure if changing `security`  from selinux to apparmor is necessary and match performance test requirement.

Reply:
Please try to use `modify_selinux_setenforce` from `lib/sles4sap.pm` at least for part of this.

Reply:
I propose not to entirely replace your implementation with just a single call to `modify_selinux_setenforce`. It looks to me that this implementation and code in `modify_selinux_setenforce` share some similar commands. Please eventually use here `modify_selinux_setenforce` for the part available in it, and keep the code for what is not implemented there.

Generally speaking, comparing the content of this PR with other existing code aroung OSADO, I can see other test modules and libs running similar commands: none is covering everything needed in this test but still I'm noticing code repetition (not in this PR but in general) and I'd like to propose any small changes possible to try yo reuse existing code.